### PR TITLE
Vitaliy Fomenko test card

### PR DESCRIPTION
**Pull request by vitaliyfomenko**
Added my card to index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+		
+		<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (Michaelin007) [inline index.html:118]: Hi, please replace "template: make a copy" with your name for both card start and card end

**Review by Michaelin007 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (vitaliyfomenko) [reply index.html]: Hi, done)

**Review by Michaelin007 — CHANGES_REQUESTED** [review]
Seen, another observation. You didn't add links to three resources. It can be your favorite programming websites. Please add useful dev resources. Thanks

File: index.html [hunk 2]
```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+		
+		<!-- ________ *Vitaliy Fomenko* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Vitaliy Fomenko</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
```
Comment (TenTraicion) [inline index.html:122]:
```suggestion
            <i class="fab fa-github"></i>
```

File: index.html [hunk 3]
```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
...
+            <a href="https://www.twitter.com/" target="_blank">Don't use Twitter</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
            <a href="https://github.com/vitaliyfomenko" target="_blank">@vitaliyfomenko</a>
```

File: index.html [hunk 4]
```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
...
+          </p>
+          <p class="about">test task</p>
```
Comment (TenTraicion) [inline index.html:125]: We don't accept any test bio. Kindly add some info about yourself here.

```suggestion
          <p class="about">About You</p>
```

File: index.html [hunk 5]
```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
...
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">no link 1</a>
```
Comment (TenTraicion) [inline index.html:130]: Add 3 resources 
replace `#` with your link 
and replace `no link` with the resource name. Adding resources are mandatory.

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@vitaliyfomenko, If you don't have twitter you can add facebook/linked in/github or other socials too. just replace `fab fa-twitter` with your desired social platform, replace the link with your user profile and handle with your user name. an example is given below. also we don't allow any test PR. so kindly add your about info. and Adding 3 resources name and link is mandatory to approve your PR. make the recommended changes and commit. 

for any kind of query you can ask here or in our [Discord Community: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF)

**Comment by CandidDeer** [comment]
@vitaliyfomenko Please note that open PRs with change requests will be closed 7 days after opening if the change requests have not been resolved by then. Thx.

**Pull request closed without merging**